src/index.js

```diff
@@ -56,6 +78,16 @@ const Spruce = {
         this.subscribers.filter(el => !! el.__x).forEach(el => {
             el.__x.updateElements(el)
         })
+    },
+
+    retrieveFromLocalStorage(name, methods = {}) {
+        const storage = JSON.parse(window.localStorage.getItem(`__spruce:${name}`))
```

Comment:
Should this whole function have a try/catch? If I remember correctly, some browsers don't let you even access localStorage in private browsing (probably Safari)

Reply:
Interesting. I'll have to take a look into that, cheers.